### DISCUSSION TITLE
v1.3.7

### DISCUSSION THREAD

**Discussion by LazyImmortal** [LazyImmortal/Sesame#6] [Announcements]
## 特别鸣谢

- 该项目复刻自[Sesame-TK](https://github.com/TKaxv-7S/Sesame-TK)，感谢作者[@TKaxv-7S](https://github.com/TKaxv-7S)对代码进行重构
- 新界面开发自[@wh-990624](https://github.com/wh-990624)，感谢作者的前端设计，由于某闭源项目违反项目开源协议，前端作者已将新UI源码闭源

## 更新说明

- 添加 抢好友大战训练项目选择项
- 添加 抢好友大战蹲点训练
- 修复 小鸡睡觉后蹲点喂食时间错误的问题

### Compatible 为兼容版本，适用于**Android 7.0**及以下的系统，最低支持**Android 5.1**

<hr /><em>This discussion was created from the release <a href='https://github.com/LazyImmortal/Sesame/releases/tag/v1.3.7'>v1.3.7</a>.</em>

**Comment by 470596** [comment]
提个小建议，能不能在更新内容里顺便备注一下开发者使用的支付宝版本，大家使用同一个版本，出现bug的情况会更少

**Reply by LazyImmortal** [reply]
我一般是有新版本就更新了的

**Comment by Neo9393** [comment]
你好，请问支付宝频繁提示访问被拒绝，请检查是否使用了代理软件或VPN是怎么回事啊？
